tests/testthat/model-examples/1.yaml

```diff
@@ -1,5 +1,5 @@
-description: original acop model
 model_type: nonmem
+description: original acop model
```

Comment:
the `model_type` and `description` fields got re-ordered by this change, which also accounts for all of the md5 hash changes below.